voir/instruments/io.py

```diff
@@ -0,0 +1,25 @@
+"""Monitor IO usage."""
+
+import psutil
+
+
+def io_monitor(perdisk=False):
+    def monitor():
+        iocounters = psutil.disk_io_counters(perdisk=perdisk)
+
+        def diskinfo(diskio):
+            return {
+                "read_count": diskio.read_count,
+                "write_count": diskio.write_count,
+                "read_bytes": diskio.read_bytes,
+                "read_time": diskio.read_time,
+                "write_time": diskio.write_time,
+                "busy_time": diskio.busy_time,
```

Comment:
```suggestion
                "busy_time": getattr(diskio, "busy_time", None),
```

This field is Linux/FreeBSD-specific, so I'd wrap it in a getattr just so that we can run this on other platforms.